app/models/concerns/turbo/broadcastable.rb

```diff
@@ -51,10 +51,10 @@ module ClassMethods
     #     belongs_to :board
     #     broadcasts_to ->(message) { [ message.board, :messages ] }, inserts_by: :prepend, target: "board_messages"
     #   end
-    def broadcasts_to(stream, inserts_by: :append, target: broadcast_target_default)
-      after_create_commit  -> { broadcast_action_later_to stream.try(:call, self) || send(stream), action: inserts_by, target: target }
-      after_update_commit  -> { broadcast_replace_later_to stream.try(:call, self) || send(stream) }
-      after_destroy_commit -> { broadcast_remove_to stream.try(:call, self) || send(stream) }
+    def broadcasts_to(stream, inserts_by: :append, target: broadcast_target_default, **rendering)
+      after_create_commit  -> { broadcast_action_later_to stream.try(:call, self) || send(stream), action: inserts_by, target: target, **rendering }
+      after_update_commit  -> { broadcast_replace_later_to stream.try(:call, self) || send(stream), **rendering }
+      after_destroy_commit -> { broadcast_remove_to stream.try(:call, self) || send(stream), **rendering }
```

Comment:
I suggest removing the rendering options for `broadcast_remove_to` that does not accept them. I don't see how that would make sense.